docs/apischema.yml

```diff
@@ -4,28 +4,79 @@ info:
   version: 0.2.0
   description: Backend server for serving PolarRoute and MeshiPhi assets
 paths:
+  /api/evaluate_route:
+    post:
+      operationId: api_route_evaluation
+      description: Provides full logging of requests and responses
+      tags:
+      - evaluate_route
+      security:
+      - {}
+      responses:
+        '200':
+          description: No response body
+  /api/mesh/{id}:
+    get:
+      operationId: api_mesh_get
+      description: Provides full logging of requests and responses
+      parameters:
+      - in: path
+        name: id
+        schema:
+          type: integer
+        required: true
+      tags:
+      - mesh
+      security:
+      - {}
+      responses:
+        '200':
+          description: No response body
+  /api/recent_routes:
+    get:
+      operationId: api_recent_routes_list
+      description: Get recent routes
+      tags:
+      - recent_routes
+      security:
+      - {}
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Route'
```

Comment:
I'm not sure this is quite correct, the response should be an array (/list) of `Route`s - though I'm not familiar enough with openapi to know what the answer is off the top of my head.